trio/tests/test_exports.py

```diff
@@ -114,6 +143,169 @@ def no_underscores(symbols):
         assert False
 
 
+# this could be sped up by only invoking mypy once per module, or even once for all
+# modules, instead of once per class.
+@slow
+# see commend on test_static_tool_sees_all_symbols
+@pytest.mark.redistributors_should_skip
+# pylint/jedi often have trouble with alpha releases, where Python's internals
+# are in flux, grammar may not have settled down, etc.
+@pytest.mark.skipif(
+    sys.version_info.releaselevel == "alpha",
+    reason="skip static introspection tools on Python dev/alpha releases",
+)
```

Comment:
I'm not sure this is actually necessary (we allow alpha releases to fail iirc!) but given `test_static_tool_sees_all_symbols` has this too, this is probably fine.

Reply:
yeah I didn't really look into why it's there and just copy-pasted it